src/main/java/com/karankumar/bookproject/backend/dto/GoodreadsBookImport.java

```diff
@@ -0,0 +1,27 @@
+package com.karankumar.bookproject.backend.dto;
```

Comment:
Could you please attach our [copyright notice](https://github.com/Project-Books/book-project/blob/master/COPYRIGHT) file to the top of this file?